include/umf/memory_pool_ops.h

```diff
@@ -17,37 +17,108 @@
 extern "C" {
 #endif
 
-/// \brief This structure comprises function pointers used by corresponding umfPool*
+///
+/// @brief This structure comprises function pointers used by corresponding umfPool*
 /// calls. Each memory pool implementation should initialize all function
 /// pointers.
+///
 typedef struct umf_memory_pool_ops_t {
     /// Version of the ops structure.
-    /// Should be initialized using UMF_VERSION_CURRENT
+    /// Should be initialized using UMF_VERSION_CURRENT.
     uint32_t version;
 
     ///
-    /// \brief Initializes memory pool.
-    /// \param providers array of memory providers that will be used for coarse-grain allocations.
+    /// @brief Initializes memory pool.
+    /// @param providers array of memory providers that will be used for coarse-grain allocations.
     ///        Should contain at least one memory provider.
-    /// \param numProvider number of elements in the providers array
-    /// \param params pool-specific params
-    /// \param pool [out] returns pointer to the pool
-    /// \return UMF_RESULT_SUCCESS on success or appropriate error code on failure.
+    /// @param numProvider number of elements in the providers array
+    /// @param params pool-specific params
+    /// @param pool [out] returns pointer to the pool
+    /// @return UMF_RESULT_SUCCESS on success or appropriate error code on failure.
+    ///
     umf_result_t (*initialize)(umf_memory_provider_handle_t provider,
                                void *params, void **pool);
 
     ///
-    /// \brief Finalizes memory pool
-    /// \param pool pool to finalize
+    /// @brief Finalizes memory pool
+    /// @param pool pool to finalize
+    ///
     void (*finalize)(void *pool);
 
-    /// Refer to memory_pool.h for description of those functions
+    ///
+    /// @brief Allocates \p size bytes of uninitialized storage of \p pool
```

Comment:
Since you fixed comments in the `memory_pool.h` (`of \p pool` -> `from \p pool`). We should fix this file in the same way.

Reply:
I've omitted this one place, thanks.